pkg/runner/cluster_swarm.go

```diff
@@ -219,7 +231,7 @@ func (*ClusterSwarmRunner) Run(ctx context.Context, input *api.RunInput, ow io.W
 				},
 			},
 			Placement: &swarm.Placement{
-				MaxReplicas: 10000,
+				MaxReplicas: 100,
```

Comment:
Changing this to 100 - originally I thought this applies to the `service` as a whole, but this is a limit per worker.

```
d21soxnh35lk        tg-dht-find-peers-4bd9c685-da5e-45d3-a93d-805d15d584b2   replicated          30/500 (max 10000 per node)   909427826938.dkr.ecr.eu-central-1.amazonaws.com/testground-dht:41fcc08e-4fb0-4d56-b11b-ac2c39d5edba
```

Having killed a few workers due to 100% CPU, I suggest we stay at 100 containers as a start before we know who much resources each container needs.